packages/forms/docs/03-fields.md

```diff
@@ -1148,6 +1148,15 @@ DateTimePicker::make('date')
     ->disabledDates(['2022-10-02', '2022-10-05', '2022-10-15'])
 ```
 
+The calendar icon can be changed, or disabled by passing `icon(false)`
```

Comment:
Maybe add a separate example below for disabling.
Also add `:` to line before example.